src/utils/fetch-config.ts

```diff
@@ -33,15 +33,19 @@ export async function fetchConfig (): Promise<void> {
 
   const businessesUrl = response.data['BUSINESSES_URL']
   sessionStorage.setItem('BUSINESSES_URL', businessesUrl)
-  console.log('Set Businesses URL to: ' + businessesUrl)
+  console.info('Set Businesses URL to: ' + businessesUrl)
```

Comment:
interesting, is the difference the format in which it displays in the console?

Reply:
They are potentially styled differently, it may be possible to filter them by type (log/info/warn/erro), and it indicates to another developer that this is informative text instead of something else.
https://developer.mozilla.org/en-US/docs/Web/API/console

Reply:
ahh ok, that is a pretty clear distinction. I like it!